### PR TITLE
Update download-software-licenses-csp.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated instructions for accessing subscription page

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7231

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/setup/download-software-licenses-csp.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ You must be a Global admin to do the steps in this article. For more information
 ## Download software and product license keys
 
 1. In the Microsoft 365 admin center, go to the **Billing** > <a href="https://go.microsoft.com/fwlink/p/?linkid=842054" target="_blank">Your products</a> page.
-2. On the **Products** tab, in the **Software** section, select the software that you want to download.
+2. On the **Products** tab, in the **Software** section, click on the product to navigate to the subscription details page.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/download-software-licenses-csp.md]:
```suggestion
2. On the **Products** tab, in the **Software** section, select the product to navigate to the subscription details page.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestion, please.

**Comment by msbemba** [comment]
@JohanFreelancer9  It works only when you click and do not select. Not sure if this can be better worded.

File: microsoft-365/admin/setup/download-software-licenses-csp.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ You must be a Global admin to do the steps in this article. For more information
 ## Download software and product license keys
 
 1. In the Microsoft 365 admin center, go to the **Billing** > <a href="https://go.microsoft.com/fwlink/p/?linkid=842054" target="_blank">Your products</a> page.
-2. On the **Products** tab, in the **Software** section, select the software that you want to download.
+2. On the **Products** tab, in the **Software** section, Click on the product to navigate to the subscription details page.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/download-software-licenses-csp.md]:
```suggestion
2. On the **Products** tab, in the **Software** section, click on the product to navigate to the subscription details page.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, thank you for the explanation. Kindly just commit the suggested change to fix the capitalization. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by CarnegieJ** [comment]
> @msbemba, thank you for the explanation. Kindly just commit the suggested change to fix the capitalization. Thanks.

The suggested change is instructional content and not spelling grammar (e.g. capitalization).

**Comment by yogkumgit** [comment]
@msbemba FYA

**Comment by msbemba** [comment]
@yogkumgit The PR is good to be merged.

File: microsoft-365/admin/setup/download-software-licenses-csp.md (hunk 1 above)
Comment (cmcatee-MSFT) [reply microsoft-365/admin/setup/download-software-licenses-csp.md]: @msbemba Please make the change suggest above for line 37.

**Comment by msbemba** [comment]
@cmcatee-MSFT The requested changes have been committed.

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please review and merge this PR. Thanks

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by tiburd** [comment]
Closing this PR.
Language complies with Microsoft style guidelines.

cc: @msbemba @yogkumgit

**Pull request closed without merging**